code/datalab-app/src/app/components/notebooks/CreateNotebookContainer.js

```diff
@@ -21,15 +23,15 @@ class CreateNotebookContainer extends Component {
     return (
       <div className='ui card'>
         <NewNotebookButton onClick={this.props.actions.openModalDialog} />
-        <Modal dimmer='blurring' open={this.props.dialogOpen}>
+        <Modal size='large' dimmer='blurring' open={this.props.dialogOpen}>
           <Modal.Header>Create a Notebook</Modal.Header>
           <Modal.Content>
-            <Grid columns={2} divided>
+            <Grid divided>
               <Grid.Row>
-                <Grid.Column>
+                <Grid.Column width={10}>
```

Comment:
Check width / column, these may differ with screen size. Should it be columns 2 & width 10, 6?

Reply:
Nice idea but sadly it turns out that isn't how semantic UI thinks about the world